src/components/Popups/PopupCursor.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import { FormattedMessage } from 'react-intl';
+import messages from './messages';
+import PopupBase from './PopupBase';
+import { Options } from './Popper';
+import './PopupCursor.scss';
+
+export type Props = {
+    isActive: boolean;
+};
+
+const options: Partial<Options> = {
+    modifiers: [
+        {
+            name: 'offset',
+            options: {
+                offset: [0, 20],
+            },
+        },
+    ],
+    placement: 'right',
+};
+
+export default class PopupCursor extends React.PureComponent<Props> {
+    popupRef = React.createRef<PopupBase>();
+
+    virtualElement = {
+        getBoundingClientRect: () => new DOMRect(0, 0, 0, 0),
+    };
+
+    constructor(props: Props) {
+        super(props);
+        document.addEventListener('mousemove', this.updatePosition);
```

Comment:
We should avoid operating on the DOM in the constructor. We should also add/remove the listener based on when the `isActive` property changes. Strongly suggest using `React.useEffect` for this.